raiden/tests/utils/transport.py

```diff
@@ -27,6 +31,51 @@
 SynapseConfigGenerator = Callable[[int], SynapseConfig]
 
 
+def new_client(server: "ParsedURL") -> GMatrixClient:
+    server_name = server.netloc
+
+    signer = make_signer()
+    username = str(to_normalized_address(signer.address))
+    password = encode_hex(signer.sign(server_name.encode()))
+
+    client = GMatrixClient(server)
+    client.login(username, password, sync=False)
+
+    return client
+
+
+def setup_broadcast_room(servers: List["ParsedURL"], broadcast_room_name: str) -> None:
+    client = new_client(servers[0])
+    room = client.create_room(alias=broadcast_room_name, is_public=True)
+
+    for server in servers[1:]:
+        client = new_client(server)
+
+        # A user must join the room to create the room in the federated server
+        room = client.join_room(room.aliases[0])
+        server_name = server.netloc
+        alias = f"#{broadcast_room_name}:{server_name}"
+
+        msg = "Setting up the room alias must not fail, otherwise the test can not run."
+        assert room.add_room_alias(alias), msg
```

Comment:
Nitpick: I''m always weary of having function calls in assert statements. They easily get overlooked and could in theory also be optimized out (probably not relevant here).

Reply:
hummm, for this one I'm not going to change since it is just test code. But if you want I can write a linter and remove for this and remove all instances from our code base.